### PR TITLE
PPA documentation language reworked

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: about/process/ppa.rst [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
+This is the package archive system for UBports projects. It hosts various PPAs containing all the DEB-components of Ubuntu Touch.
 
 Repository naming convention
 ----------------------------
 
 Native packages
 ^^^^^^^^^^^^^^^
 
-Native packages are repositories which contain a ``debian/`` directory **with** the source used to create the Debian source package (``.dsc``).
+Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the Debian source package (``.dsc``).
```
Comment (comradekingu) [inline about/process/ppa.rst:12]: This is what it is with regular Debian packages anyway.

File: about/process/ppa.rst [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
+The repository should have the name of the source package it generates with ``-packaging`` appended to the end.
+For example, a packaging repository that generates a source package called ``sandwich`` would be called ``sandwich-packaging``.
 
 Creating new PPAs
 -----------------
 
 New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`. The name of the branch translates literally to the name of the PPA: ``http://repo.ubports.com/dists/[branch name]``
 
-Non-standard PPAs (i.e. not ``xenial``, ``vivid`` or ``bionic``) are kept for three months. In case they need to be kept for a longer time, a file with the name ``ubports.keep`` can be created in the root of the repository, containing the date until which the PPA shall be kept open in the form of ``YYYY-MM-dd``. If this file is empty, the PPA will be kept for two years after the last build.
+Non-standard PPAs (i.e. not ``vivid`, ``xenial``, ` or ``bionic``) are kept for three months.
```
Comment (comradekingu) [inline about/process/ppa.rst:30]:
```suggestion
Non-standard PPAs (i.e. not ``vivid``, ``xenial``, or ``bionic``) are kept for three months.
```

Comment (comradekingu) [inline about/process/ppa.rst]:
```suggestion
Non-standard PPAs (i.e. not ``vivid`, ``xenial``, or ``bionic``) are kept for three months.
```

File: about/process/ppa.rst [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
+For example, a packaging repository that generates a source package called ``sandwich`` would be called ``sandwich-packaging``.
 
 Creating new PPAs
 -----------------
 
 New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`. The name of the branch translates literally to the name of the PPA: ``http://repo.ubports.com/dists/[branch name]``
 
-Non-standard PPAs (i.e. not ``xenial``, ``vivid`` or ``bionic``) are kept for three months. In case they need to be kept for a longer time, a file with the name ``ubports.keep`` can be created in the root of the repository, containing the date until which the PPA shall be kept open in the form of ``YYYY-MM-dd``. If this file is empty, the PPA will be kept for two years after the last build.
+Non-standard PPAs (i.e. not ``vivid``, ``xenial``, or ``bionic``) are kept for three months.
+If they need to be kept for longer, a file with the name ``ubports.keep`` containing a date in the form of in the form of ``YYYY-MM-dd` can be used to decide how long to keep the PPA in the repository for.
```
Comment (comradekingu) [inline about/process/ppa.rst:31]:
```suggestion
If they need to be kept for longer, a file with the name ``ubports.keep`` containing a date in the form of in the form of ``YYYY-MM-dd`` can be used to decide how long to keep the PPA in the repository for.
```

File: about/process/ppa.rst [hunk 4]
```diff
@@ -1,28 +1,32 @@
...
+Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the Debian source package (``.dsc``).
 
-The name of the Debian source package generated from the repository and the name of the git-repository should be the same.
+The name of the Debian source package generated from the repository and the name of the Git-repository should be the same.
 
 Packaging repositories
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Packaging repositories contain a ``debian/`` directory **without** the source used to create the Debian source package (``.dsc``). They also contain instructions to tell ``debhelper`` how to get the source used to create the source package.
+Packaging repositories contain a ``debian/`` directory **without** the source used to create the Debian source package (``.dsc``).
```
Comment (comradekingu) [inline about/process/ppa.rst:19]:
```suggestion
Packaging repositories contain a ``DEBIAN`` folder **without** the source used to create the Debian source package (``.dsc``).
```

File: about/process/ppa.rst (hunk 1 above)
Comment (comradekingu) [inline about/process/ppa.rst:12]:
```suggestion
Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the DEB source package (``.dsc``).
```

File: about/process/ppa.rst [hunk 5]
```diff
@@ -1,28 +1,32 @@
...
+They also contain instructions to tell ``debhelper`` how to get the sources used to create the source package.
 
-The repository should have the name of the source package it generates with ``-packaging`` appended to the end. For example, a packaging repository that generates a source package called ``sandwich`` would be called ``sandwich-packaging``.
+The repository should have the name of the source package it generates with ``-packaging`` appended to the end.
+For example, a packaging repository that generates a source package called ``sandwich`` would be called ``sandwich-packaging``.
 
 Creating new PPAs
 -----------------
 
 New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`. The name of the branch translates literally to the name of the PPA: ``http://repo.ubports.com/dists/[branch name]``
```
Comment (comradekingu) [inline about/process/ppa.rst:28]:
```suggestion
New PPAs can be created dynamically by the CI server using a special :doc:`git-branch naming convention <branch-naming>`. The name of the branch translates literally to the name of the PPA: ``https://repo.ubports.com/dists/[branch name]``
```

File: about/process/ppa.rst [hunk 6]
```diff
@@ -1,28 +1,32 @@
...
+This is the package archive system for UBports projects. It hosts various PPAs containing all the DEB-components of Ubuntu Touch.
 
 Repository naming convention
 ----------------------------
 
 Native packages
 ^^^^^^^^^^^^^^^
 
-Native packages are repositories which contain a ``debian/`` directory **with** the source used to create the Debian source package (``.dsc``).
+Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the DEB source package (``.dsc``).
```
Comment (doniks) [inline about/process/ppa.rst:12]: lower case I think e.g., https://github.com/ubports/system-settings also below

File: about/process/ppa.rst [hunk 7]
```diff
@@ -1,28 +1,32 @@
...
+Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the Debian source package (``.dsc``).
 
-The name of the Debian source package generated from the repository and the name of the git-repository should be the same.
+The name of the Debian source package generated from the repository and the name of the Git-repository should be the same.
 
 Packaging repositories
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Packaging repositories contain a ``debian/`` directory **without** the source used to create the Debian source package (``.dsc``). They also contain instructions to tell ``debhelper`` how to get the source used to create the source package.
+Packaging repositories contain a ``DEBIAN`` folder **without** the source used to create the Debian source package (``.dsc``).
```
[the remark this answers is not in the document]
Comment (doniks) [reply about/process/ppa.rst:19]: like this I think https://github.com/ubports/pulseaudio-packaging . maybe good to add these examples acutally

**Review by doniks — CHANGES_REQUESTED** [review]

File: about/process/ppa.rst (hunk 6 above)
Comment (comradekingu) [inline about/process/ppa.rst:12]:
```suggestion
Native packages are repositories which contain a ``debian/`` folder **with** the source used to create the DEB source package (``.dsc``).
```

File: about/process/ppa.rst [hunk 8]
```diff
@@ -1,28 +1,32 @@
...
+Native packages are repositories which contain a ``DEBIAN/`` folder **with** the source used to create the DEB source package (``.dsc``).
 
-The name of the Debian source package generated from the repository and the name of the git-repository should be the same.
+The name of the Debian source package generated from the repository and the name of the Git-repository should be the same.
 
 Packaging repositories
 ^^^^^^^^^^^^^^^^^^^^^^
 
-Packaging repositories contain a ``debian/`` directory **without** the source used to create the Debian source package (``.dsc``). They also contain instructions to tell ``debhelper`` how to get the source used to create the source package.
+Packaging repositories contain a ``DEBIAN`` folder **without** the source used to create the Debian source package (``.dsc``).
```
Comment (comradekingu) [inline about/process/ppa.rst:19]:
```suggestion
Packaging repositories contain a ``debian`` folder **without** the source used to create the Debian source package (``.dsc``).
```

**Review by doniks — APPROVED** [review]

**Pull request merged**